### PR TITLE
Added template kalpa-hdinsight

### PR DESCRIPTION
**Pull request by mukulkgupta**
### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use resourceGroup().location for resource locations
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/bp-checklist.md

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### Changelog

*
*
*

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@mukulkgupta__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by azurecla** [comment]
Hi __@mukulkgupta__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by azurecla** [comment]
__@mukulkgupta__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, AZPRBOT;

**Comment by gatneil** [comment]
Is this a duplicate of https://github.com/Azure/azure-quickstart-templates/pull/2917?

**Comment by mukulkgupta** [comment]
@gatneil ,  Yes it was done by mistake , I have closed the #2917

File: kalpa-hdinsight/Documents/activity guide.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
+## Activity 1: Understand & Process Real-time data.
```
Comment (singhkays) [inline kalpa-hdinsight/Documents/activity guide.md]: no spaces in the filenames please

File: kalpa-hdinsight/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,473 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {},
+    "variables": {
+        "KalpaNamespaceName": "[concat('ns',uniquestring(resourceGroup().id))]",
```
Comment (singhkays) [inline kalpa-hdinsight/azuredeploy.json]: camelCase parameter names

File: kalpa-hdinsight/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,473 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json#",
```
Comment (singhkays) [inline kalpa-hdinsight/azuredeploy.json]: Use 2015-01-01 for schema version

**Comment by gatneil** [comment]
no activity; closing

**Pull request closed without merging**